sklearn/mixture/gmm.py

```diff
@@ -501,6 +533,26 @@ def fit(self, X, y=None):
             self.covars_ = best_params['covars']
             self.means_ = best_params['means']
             self.weights_ = best_params['weights']
+
+        return responsibilities
+
+    def fit(self, X, y=None):
+        """Estimate model parameters with the expectation-maximization
+        algorithm.
```

Comment:
style: this should fit in one line. For instance use:

```
""""Estimate model parameters with the EM algorithm

...
"""
```

then mention the Expectation Maximization in the detailed paragraph.